doc/docs.md

```diff
@@ -789,9 +789,8 @@ if a < b {
 }
 ```
 
-`if` statements are pretty straightforward and similar to most other languages.
-Unlike other C-like languages,
-there are no parentheses surrounding the condition and the braces are always required.
+`if` statements are pretty straightforward and similar to most other languages.   
+Unlike C or other C-like languages, there are no parentheses surrounding the condition and the braces are always required.
```

Comment:
The line is too long. It also does not provide any improvement over the existing sentence imho.